docs/source/api_reference/transformations.rst

```diff
@@ -393,6 +393,17 @@ Datetime feature generation
 
     DateTimeFeatures
 
+Trigonometric Features
```

Comment:
I would say spectral features or frequency features or similar. Trigonometric is easy to confuse with the category of cosine transform etc

Reply:
I think we should stick to "Trigonometric terms" as it's the term used consistently in the literature. 

Here are 2 of the earlier papers I could find:

1. Harvey, A., Koopman, S. J. & Riani, M. (1997), ‘The modeling and seasonal adjustment of 558 weekly observations’, Journal of Business and Economic Statistics 15, 354–368.
2. Harvey, A. & Koopman, S. J. (1993), ‘Forecasting hourly electricity demand using time556 varying splines.’, Journal of the American Statistical Association 88, 1228–1236.

and here is the TBATS paper: 
1. [De Livera, A.M., Hyndman, R.J., & Snyder, R. D. (2011), Forecasting time series with complex seasonal patterns using exponential smoothing, Journal of the American Statistical Association, 106(496), 1513-1527.](https://robjhyndman.com/papers/ComplexSeasonality.pdf#page=37&zoom=100,76,825)

Reply:
how about "seasonality and calendar features", and put it together with the date/time transformer?

Reply:
OK - that makes sense